templates/etc/quagga/debian.conf.j2

```diff
@@ -1,9 +1,12 @@
 #
 vtysh_enable=yes
-zebra_options=" --daemon -A 127.0.0.1 -P 2601 -u {{ quagga_user }} -g quagga --keep_kernel --retain"
+zebra_options=" --daemon -A 127.0.0.1 -P 2601 -u {{ quagga_user }} -g quagga --keep_kernel --retain {{ quagga_zebra_options }}"
 bgpd_options=" --daemon -A 127.0.0.1 -P 2605 -u {{ quagga_user }} -g quagga --retain -p 179"
 ospfd_options=" --daemon -A 127.0.0.1 -P 2604 -u {{ quagga_user }} -g quagga"
 ospf6d_options=" --daemon -A ::1 -P 2606 -u {{ quagga_user }} -g quagga"
 ripd_options=" --daemon -A 127.0.0.1 -P 2602 -u {{ quagga_user }} -g quagga --retain"
 ripngd_options=" --daemon -A ::1 -P 2603 -u {{ quagga_user }} -g quagga --retain"
 isisd_options=" --daemon -A 127.0.0.1 -P 2608 -u {{ quagga_user }} -g quagga"
+
+watchquagga_enable=yes
+watchquagga_options=(-adz -r /usr/sbin/servicebBquaggabBrestartbB%s -s /usr/sbin/servicebBquaggabBstartbB%s -k /usr/sbin/servicebBquaggabBstopbB%s -b bB -t 90)
```

Comment:
Double- or triple-underscore would be much more readable than "bB".  Or maybe we can do away with the "-b" hack entirely, going by comments in /etc/quagga/debian.conf from router2.common.sfdc:

    brian.strand@router2:~$ sudo cat /etc/quagga/debian.conf
    #
    # If this option is set the /etc/init.d/quagga script automatically loads
    # the config via "vtysh -b" when the servers are started. 
    # Check /etc/pam.d/quagga if you intend to use "vtysh"!
    #
    vtysh_enable=yes
    zebra_options="  --daemon -A 127.0.0.1"
    bgpd_options="   --daemon -A 127.0.0.1"
    ospfd_options="  --daemon -A 127.0.0.1"
    ospf6d_options=" --daemon -A ::1"
    ripd_options="   --daemon -A 127.0.0.1"
    ripngd_options=" --daemon -A ::1"
    isisd_options="  --daemon -A 127.0.0.1"
    babeld_options=" --daemon -A 127.0.0.1"
    #
    # Please note that watchquagga_options is an array and not a string so that
    # quotes can be used.
    #
    # The list of daemons to watch is automatically generated by the init script
    # from daemons.conf and appended to the watchquagga_options.
    # Example:
    #    watchquagga_options=("-Adz" "-r" '/sbin/service %s restart' -s '/sbin/service %s start'  -k '/sbin/service %s stop')
    watchquagga_enable=yes
    watchquagga_options=(--daemon)

Also a 90-second timeout seems really long; does bgpd need this when it's taking full Internet routing tables or something?